site/_config.yml

```diff
@@ -1,17 +1,17 @@
 title: Apache BookKeeper
-subtitle: A replicated log service
+subtitle: A scalable, fault tolerant and low latency storage serivce optimized for realtime workloads
```

Comment:
A scalable, fault-tolerant, and low-latency storage service optimized for real-time workloads